### PR TITLE
[CFM-474] notify group admins that status has changed to archived

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 

https://issuetracker.amplexor.com/jira/browse/CFM-474

### CODE REVIEW

**Comment by anvmn** [comment]
selection_110

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -80,9 +80,31 @@ function c4m_user_notifications_group_status_set_unpublished($node, $group_forme
...
+ */
+function _c4m_user_notifications_to_admin_group_status_set_archived($node, $admin_id, $group_former_status) {
+  $message = message_create('c4m_group_archived', array('uid' => $admin_id));
+  c4m_message_load_tokens__c4m_group_archived($message);
+  $message->arguments['@group_former_status'] = $group_former_status;
+
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $wrapper->field_group_node->set($node->nid);
+  // Message will be saved in message_notify_send_message().
+  message_notify_send_message($wrapper->value());
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: `message_subscribe_send_message()`

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module [hunk 2]
```diff
@@ -78,6 +79,20 @@ function _c4m_message_format_welcome_to_group_from_field(&$message, $message_ent
 /**
  * Helper function.
  *
+ * Formats 'c4m_group_archived' message From field, per template requirement.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @anvmn 
Can you please write the ID of the template ?
thanks

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 3]
```diff
@@ -25,6 +25,33 @@ function c4m_message_views_api($module = NULL, $api = NULL) {
...
+          "format" : "full_html",
+          "safe_value" : "\\u003Cp\\u003EDear @group_admin_full_name,\\u003C\\/p\\u003E\\n\\u003Cp\\u003E\\n\\u003Ca href=\\u0022!group_dashboard_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@group_node_title\\u003C\\/a\\u003E is now archived.\\u003C\\/p\\u003E\\n\\u003Cp\\u003E@event_user_full_name updated the status of this @group_node_type from @group_former_status to Archived.\\u003C\\/p\\u003E\\n\\u003Cp\\u003EYou can consult \\u003Ca href=\\u0022url]help-guidance\\u0022 rel=\\u0022nofollow\\u0022\\u003Ethis help section\\u003C\\/a\\u003E to find out more about Group statuses and what they imply.\\u003C\\/p\\u003E\\n\\u003Cp\\u003E\\nThe capacity4dev team\\u003C\\/p\\u003E\\n"
```
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: Token shows status in all-lowercase. Resulting in: "... updated the status of the group from published to Archived".
Please make this consistent.

Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: Token is not correct:

screen shot 2016-09-01 at 09 22 22

Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: "find out more about Group statuses and what they imply."
Please include the group type token here, so it works for projects as well.

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: Regarding `help section` token, which appears at other templates as well:
Notification templates document doesn't specify where the link should point to.
Can you provide this info?

Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: Help section is at `/help-guidance/help-guidance`

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**